docs/developers/quickstart.mdx

```diff
@@ -156,13 +152,142 @@ contract HelloSource {
+    process.exit(0);
+  } catch (error) {
+    console.log(error);
+    process.exit(1);
+  }
+};
+
+runMain();
+```
+
+Then call the deploy script: 
+```bash
+$ npx hardhat run scripts/deploy.js --network goerli
+Contract deployed to: 0x8CC1DB2a76ea4bc40089f0Db2B25f8B13032F72d
+```
+
+Note the contract address because you'll want to verify your contracts later.
+
+## Target Contract
+
+Follow the same steps above to create a new hardhat project, this time name the project `GreetingDestination.` Install all the same dependencies as above. 
+
+All target contracts must implement Connext's `IXReceiver` interface. This interface ensures that Connext can call the contract and pass necessary data. 
+
+```solidity showLineNumbers
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.15;
+
+import {IXReceiver} from "@connext/nxtp-contracts/contracts/core/connext/interfaces/IXReceiver.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract HelloTarget is IXReceiver {
+  string public greeting;
+
+  // Hardcoded cost to update the greeting, in wei units
+  uint256 public cost = 1e18;
+
+  // The TEST Token on Mumbai
+  IERC20 public token = IERC20(0xeDb95D8037f769B72AAab41deeC92903A98C9E16);
+
+  /** @notice The receiver function as required by the IXReceiver interface.
+    * @dev The Connext bridge contract will call this function.
+    */
+  function xReceive(
+    bytes32 _transferId,
+    uint256 _amount,
+    address _asset,
+    address _originSender,
+    uint32 _origin,
+    bytes memory _callData
+  ) external returns (bytes memory) {
+    // Enforce the cost to update the greeting
+    require(
+      _asset == address(token) && _amount >= cost,
+      "Must pay at least 1 TEST"
+    );
+
+    // Unpack the _callData
+    string memory newGreeting = abi.decode(_callData, (string));
+
+    _updateGreeting(newGreeting);
+  }
+
+  /** @notice Internal function to update the greeting.
+    * @param newGreeting The new greeting.
+    */
+  function _updateGreeting(string memory newGreeting) internal {
+    greeting = newGreeting;
+  }
+}
+```
+
+With this code, you'll be able to update the greeting from the source contract.
+
 ## Executing the Transaction
 
+Now that you've deployed your contracts, let's verify that we're actually able to access the source contract's functionality from the destination contract.
+
 If you don't already have gas funds on Goerli, try these faucets to get some:
 - https://goerli-faucet.mudit.blog/ (Requires Twitter account)
 - https://goerlifaucet.com/ (Requires signing up with Alchemy)
 
-You should try deploying (and verifying) your own contracts, but you can use contracts we already deployed for this example:
+For the following steps, you should try deploying (and verifying) your own contracts. For ease, you can use the contracts we deployed for this part:
 - [`HelloSource.sol`](https://goerli.etherscan.io/address/0x9ce3799f033d89d316f373f1db161c84a401a26c) 
```

Comment:
since we're changing the contract names, it would be great to redeploy these as well. we will need to anyways since the Connext contracts were updated

Reply:
redeployed and updated the contracts in the quickstart!